src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -73,11 +106,11 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
 
     @Test
     public void execute_noFieldSpecifiedUnfilteredList_success() {
-        EditEntryDescriptor descriptor = new EditEntryDescriptor();
-        descriptor.setTag(null);
-        descriptor.setType(new EntryType("e"));
-        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
-        Entry editedEntry = model.getFilteredExpenditureList().get(INDEX_FIRST_PERSON.getZeroBased());
+        EditEntryDescriptor descriptor = new EditEntryDescriptorBuilder()
```

Comment:
Similarly, this test specifies no fields at all, so we create an empty `EditEntryDescriptor` with just `EntryType`.

Reply:
Hey, thanks for this, I remember I faced some issues with creating the descriptor without any `Tag`s at all, but I could have mixed it up with another test case I was working on.